test/src/API/templates/templates.test.js

```diff
@@ -37,24 +37,19 @@ chai.use(deepEqualInAnyOrder);
 chai.use(chaiResValidator(pathToApiSpec));
 
 describe('Template API tests', function() {
+    // Set the default timeout for all tests
+    this.timeout(testTimeout.short);
     // Save the state of the repos before any test and restore the exact state after
     saveReposBeforeTestAndRestoreAfter();
     describe('GET /api/v1/templates', function() {
-        before(async function() {
-            this.timeout(testTimeout.short);
-            await setTemplateReposTo([
-                { ...sampleRepos.codewind },
-            ]);
-        });
+        setupReposAndTemplatesForTesting();
         describe('?projectStyle=', function() {
             it('should return at least the default Codewind templates as no projectStyle is given', async function() {
-                this.timeout(testTimeout.short);
```

Comment:
are you sure this is always OK in Jenkins, and never times out when getting templates?

Reply:
Oops, see you're setting it for all tests. Nice 👍